source/includes/administration/_users.md

```diff
@@ -329,23 +329,45 @@ Optional | &nbsp;
 
 The responses' `data` field contains the updated [user](#administration-users).
 
-
 <!-------------------- DELETE USER -------------------->
 
-
 ### Delete user
 
 `DELETE /users/:id`
 
+Users are deleted asynchronously on the underlying services.
+
+The user model's status will be set to `DISABLED` prior to attempting the delete operation. This will prevent the user from login in or making API calls.
+
+The user model will be deleted only when the user is removed from all environments AND is deleted on each underlying service.
```

Comment:
**Suggestion**
Could be: ` AND is deleted on each underlying` --> ` AND from all underlying services`

Reply:
Done